app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -31,62 +43,56 @@ fun DetailRoute(
 fun DetailScreen(modifier: Modifier = Modifier, onClickToList: () -> Unit, id: String) {
     var cardFace by remember { mutableStateOf(CardFace.FRONT) }
 
-    Box(
+    Column(
         modifier = modifier
             .fillMaxSize()
-            .wrapContentSize()
-            .size(192.dp, 240.dp)
+            .background(CategoryLevelDummy.valueOf(level).backgroundColor)
     ) {
-        TestFlipCard(cardFace, onClickToList, id)
+        DetailHeader(cardFace = cardFace, onBackClick = { cardFace = CardFace.BACK })
+        Box(modifier = modifier.fillMaxSize()) {
+            DetailFlipCard(Modifier.align(Alignment.Center), cardFace, onClickToList, id)
+        }
     }
+}
 
+@Composable
+fun DetailHeader(cardFace: CardFace, onBackClick: () -> Unit) {
     if (cardFace == CardFace.FRONT) {
-        Button(
-            onClick = {
-                cardFace = CardFace.BACK
-            }, modifier = modifier.wrapContentSize()
-        ) {
-            Text(text = "Back")
-        }
+        Icon(
+            modifier = Modifier
+                .padding(
+                    horizontal = dimensionResource(id = R.dimen.horizontal_padding),
+                    vertical = dimensionResource(id = R.dimen.vertical_padding)
+                )
+                .size(24.dp)
+                .clickable { onBackClick() },
+            painter = painterResource(id = R.drawable.ic_arrow_left),
+            contentDescription = null,
+        )
     }
 }
 
 @Composable
-fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
+fun DetailFlipCard(
+    modifier: Modifier = Modifier, cardFace: CardFace, onClickToList: () -> Unit, id: String
```

Comment:
🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)